tests/main/views/test_marketplace.py

```diff
@@ -607,6 +610,209 @@ def test_supplier_applied_view_application_for_closed_opportunity(self):
+
+        expected_text = "Log in to view question and answer session details"
+        expected_link = '/suppliers/opportunities/{}/question-and-answer-session'.format(self.brief_id)
+
+        assert document.xpath('.//a[contains(text(),"{}")]'.format(expected_text))[0].text
+        assert document.xpath('.//a[contains(text(),"{}")]'.format(expected_text))[0].attrib['href'] == expected_link
+
+    def test_live_brief_q_and_a_session_logged_in(self):
+        """
+        As long as a:
+            Supplier user is logged in
+            The brief is live
+            Clarification questions are open
+            The brief has Q and A session details
+        We should show the:
+            Link to view the QAS details
+        """
+        self.login_as_supplier()
+
+        res = self.client.get('/digital-outcomes-and-specialists/opportunities/{}'.format(self.brief_id))
+        document = html.fromstring(res.get_data(as_text=True))
+
+        assert res.status_code == 200
+
+        expected_text = "View question and answer session details"
+        expected_link = '/suppliers/opportunities/{}/question-and-answer-session'.format(self.brief_id)
+
+        assert document.xpath('.//a[contains(text(),"{}")]'.format(expected_text))[0].text
+        assert document.xpath('.//a[contains(text(),"{}")]'.format(expected_text))[0].attrib['href'] == expected_link
+
+    @pytest.mark.parametrize(
+        'brief_data', [
+            {'status': 'withdrawn'},
+            {'status': 'closed'},
+            {'questionAndAnswerSessionDetails': None},
+            {'clarificationQuestionsAreClosed': True}
+        ]
+    )
+    def test_brief_q_and_a_session_fails(self, brief_data):
+        """
+        On viewing briefs with data like the above the page should load but we should not get the link.
+        """
+        self.brief['briefs'].update(brief_data)
+
+        res = self.client.get('/digital-outcomes-and-specialists/opportunities/{}'.format(self.brief_id))
+        document = html.fromstring(res.get_data(as_text=True))
+
+        assert res.status_code == 200
+
+        unexpected_texts = [
+            "Log in to view question and answer session details",
+            "View question and answer session details"
+        ]
+        for unexpected_text in unexpected_texts:
+            with pytest.raises(IndexError):
+                document.xpath('.//a[contains(text(),"{}")]'.format(unexpected_text))[0].text
+
+
+class TestBriefPageQandASectionAskAQuestion(BaseBriefPageTest):
+
+    def setup_method(self, method):
+        super(TestBriefPageQandASectionAskAQuestion, self).setup_method(method)
+        self.brief['briefs']['clarificationQuestionsAreClosed'] = False
+
+    def test_live_brief_ask_a_question(self):
+        """
+        As long as a:
+            A user is not logged in
+            The brief is live
+            Clarification questions are open
+            The brief has Q and A session details
```

Comment:
The brief does not need to have Q and A session details for questions to be asked